source/includes/_getting_started.md

```diff
@@ -25,6 +25,18 @@ You must replace <code>your_api_key</code> with your personal API key.
 
 The API key is found from the API keys section under the user profile menu. If you don't see CloudMC API keys section, contact your system administrator as you may not have the permission to see that section. **Your API key carries the same privileges as your CloudMC account, so be sure to keep it secret**. If you think your API has been compromised, regenerate your API key from the API keys section.
 
+## Working with sub-organizations
+
+*If you don't know what a sub-organization is, you can safely skip this section.*
+
+When listing entities or carrying out an operation in an organization other than your own, make sure to specify the `org_id` query parameter in your request. For operations, this looks like:
+
+<code>https://cloudmc_endpoint/api/v1/services/:service_code/:environment_name/:entity_type/:entity_id?operation=:operation&<strong>org_id=:org_id</strong></code>
```

Comment:
Since it's in the Getting Started section now, how about using a simpler API endpoint in this example?

Reply:
How does something like this sound:
(and removing lines 36, 38)

``` 
When using the API in an organization other than your own, make sure to specify the `org_id` query parameter in your request. This looks like:

<code>https://cloudmc_endpoint/api/v1/users&<strong>org_id=:org_id</strong></code>```